### DISCUSSION TITLE
Octoprint Setup - Teddy Warner

### DISCUSSION THREAD

**Discussion by giscus** [Twarner491/theodore.net#7] [General]
# Octoprint Setup - Teddy Warner

Project Portfolio and Documentation Site for Teddy Warner

https://teddywarner.org/Projects/Octoprint/

**Comment by JoseMoranRodriguez** [comment]
This setup is wonderful, great job!!

**Reply by Twarner491** [reply]
Thanks! Glad you found it helpful!!
